default-fields/0000-default-fields.md

```diff
@@ -6,19 +6,224 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow struct definitions to supply default values for individual fields,
+and then allow those fields to be omitted from construction:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32
+}
+
+let foo = Foo {
+    a: "Hello",
+    c: 42
+}
+```
 
 # Motivation
 [motivation]: #motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Today, Rust allows you to create an instance of a struct using a literal syntax:
+
```

Comment:
Is there a standard terminology for this syntax? Some languages call them "struct literals", maybe we can call them "struct initialisers", the reference uses "struct expressions".